operator/cmd/root.go

```diff
@@ -95,6 +96,14 @@ var (
 			registerOperatorHooks,
 		),
 
+		resourcesCell,
```

Comment:
`resourcesCell` and `lbipam.Cell` should probably be within `WithLeaderLifecycle`?